### DISCUSSION TITLE
`w-{fraction}` is not work in stylus @apply

### DISCUSSION THREAD

**Discussion by x-ray-s** [tailwindlabs/tailwindcss#4177] [General]
In Stylus if the input is
```
.block {
  @apply w-1/4;
}
```

The output will be 

```
...
  @apply w-1 / 4;
```

The CSS syntax identifiers can contain only the characters [_-a-zA-Z0-9]，the `/` will be parsed as an arithmetic operator.
 `w-1` is ident, `/` is literal, so I got an error output.
Suggest supporting an alias, like w-1_4.

**Comment by adamwathan** [comment]
Hey! You'll have to use the `@css` directive to tell Stylus not to process it:

https://tailwindcss.com/docs/using-with-preprocessors#stylus

No plans to make changes to Tailwind itself to support specific preprocessors unfortunately, we really recommend just using PostCSS:

https://tailwindcss.com/docs/using-with-preprocessors#using-post-css-as-your-preprocessor
